internal/provider/schemas/team.go

```diff
@@ -67,3 +78,85 @@ func TeamDataSourceSchemaAttributes() map[string]datasourceSchema.Attribute {
 		},
 	}
 }
+
+func TeamResourceSchemaAttributes() map[string]resourceSchema.Attribute {
+	return map[string]resourceSchema.Attribute{
+		"id": resourceSchema.StringAttribute{
+			MarkdownDescription: "Team ID",
+			Computed:            true,
+			PlanModifiers: []planmodifier.String{
+				stringplanmodifier.UseStateForUnknown(),
+			},
+		},
+		"name": resourceSchema.StringAttribute{
+			MarkdownDescription: "Team name",
+			Required:            true,
+		},
+		"description": resourceSchema.StringAttribute{
+			MarkdownDescription: "Team description",
+			Optional:            true,
+		},
+		"member_ids": resourceSchema.SetAttribute{
+			ElementType:         types.StringType,
+			MarkdownDescription: "The IDs of the users to add to the Team",
+			Optional:            true,
+			Validators: []validator.Set{
+				setvalidator.ValueStringsAre(validators.IsCuid()),
+			},
+		},
+		"is_idp_managed": resourceSchema.BoolAttribute{
+			MarkdownDescription: "Whether the Team is managed by an identity provider",
+			Computed:            true,
+		},
+		"organization_id": resourceSchema.StringAttribute{
```

Comment:
i don't think we need this
we should use r.OrganizationId in `Create` and `Update` - they should exist there at runtime